pkgs/applications/search/recoll/default.nix

```diff
@@ -55,6 +55,11 @@ mkDerivation rec {
   ] ++ lib.optional withGui qtbase
     ++ lib.optional stdenv.isDarwin libiconv;
 
+  # if /etc/debian_version exists, recoll passes the unsupported --install-layout=deb option to setup.py
+  preInstall = ''
```

Comment:
Exists that file in the source tarball? If so we should do this in postpatch.

Reply:
I meant if the Makefile is already in the source tarball. Patching should normally be done after we have run patchPhase.

Reply:
Oh, I get it.  Sorry, now I understand.  Fixed in latest push.